module-services/service-cellular/ServiceCellular.cpp

```diff
@@ -608,7 +605,7 @@ void ServiceCellular::registerMessageHandlers()
         auto message = static_cast<cellular::SwitchVolteOnOffRequest *>(request);
         auto channel = cmux->get(CellularMux::Channel::Commands);
         if (channel == nullptr) {
-            LOG_ERROR("[VoLTE] failed to get channel, skipping request");
+            LOG_ERROR("Skipping VoLte request, failed to get channel");
```

Comment:
Use `VoLTE` as before

Reply:
I've talked with Marcin about it, and we decided not to put tags like this in logs. Also, these VoLte logs don't seem to be very important.